element-hq/element-x-android
# Use the right colors for `@room` mention pills

## Content

Since an `@room` mentions will also implicitly mention the current use, it should use the same color as an explicit mention to the current user.

## Motivation and context

Fixes #3374.

## Screenshots / GIFs

|Before|After|
|-|-|
|![](https://media.githubusercontent.com/media/element-hq/element-x-android/08a6b083eaf0e86a9bd24248ea9f94924d60d347/tests/uitests/src/test/snapshots/images/libraries.textcomposer.mentions_MentionSpanTheme_Day_0_en.png)|![](https://media.githubusercontent.com/media/element-hq/element-x-android/d159136b25337c478f07c1f74249d29a12c3014f/tests/uitests/src/test/snapshots/images/libraries.textcomposer.mentions_MentionSpanTheme_Day_0_en.png)|

## Tests



- Open a room with an `@room` mention.
- Check the right colors are used.

## Tested devices

- OS version(s): 14

## Checklist
